UI/CMakeLists.txt

```diff
@@ -20,6 +20,10 @@ if(NOT TARGET OBS::json11)
   add_subdirectory("${CMAKE_SOURCE_DIR}/deps/json11" "${CMAKE_BINARY_DIR}/deps/json11")
 endif()
 
+if(NOT TARGET OBS::qt-wrappers)
```

Comment:
This block (as well as adding the library target as a dependency) should be moved into `ui-qt.cmake` for encapsulation.

Reply:
Done